test/integration/testsForApiVersion1.spec.js

```diff
@@ -0,0 +1,111 @@
+const { setupForApiVersion1 } = require('./util')
+const evrythng = require('../../dist/evrythng.node')
+
+describe('evrythng.js for apiVersion = 1', function () {
+  let settings = evrythng.setup({ apiVersion: 1, geolocation: false })
+  const { apiUrl, apiVersion } = settings
+
+  before(async () => {
+    settings = await evrythng.setup({ apiVersion: 1, geolocation: false })
+    await setupForApiVersion1(apiUrl)
+  })
+
+  describe(`as Application for API ${apiVersion}`, () => {
```

Comment:
nit: `API v${apiVersion}` including `v` for version, and similar for other `describe`s

Reply:
fixed, thanks!